.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,45 @@
+name: "💡 Feature Request"
+description: Request an ideas for this project
+title: "💡 [REQUEST] - <title>"
+labels: [
+  "Feature Proposal"
+]
+body:
+  - type: checkboxes
+    attributes:
+      label: Does the feature exist in the most recent commit?
+      options:
+        - label: It doesn't exist.
```

Comment:
I think this checkbox can help double checking if the feature is already applied or in review. Good attribute! 👍

Reply:
> Check my reviews :)

> I think this checkbox can help double checking if the feature is already applied or in review. Good attribute! 👍

Thanks a lot!  It is so helpful to me!